src/stellar/stellarEngine.js

```diff
@@ -533,7 +538,7 @@ export class StellarEngine extends CurrencyEngine {
       this.otherData.accountSequence++
       this.walletLocalDataDirty = true
     } catch (e) {
-      this.log(e)
+      this.log.error(e)
```

Comment:
Add a line that logs the fact that you are in broadcastTx. ie

  this.log.error(`ERROR broadcastTx ${JSON.stringify(e)}`)